roles/build_openstack_packages/templates/projects.ini.j2

```diff
@@ -28,10 +28,11 @@ use_components={{ cifmw_bop_use_components }}
 # There default values are `0.date.hash` and `0` respectively.
 # We are only setting these for downstream case when cifmw_bop_osp_release
 # is defined. For downstream case, we are hardcoding release_numbering=minor.date.hash
-# and then just set the release_minor value based on the OSP version
-# (e.g. 0 for 18/18.0, 1 for 18.1, 2 for 18.2 etc).
+# and then just set the release_minor value based on the OSP version.
+# (e.g. 18.0 for rhos-18.0).
```

Comment:
i guess this kidn of explains why you are doing this but its clearer form the commit message

"The correct value for release_minor is the rhos-release version i.e. for RHOS-18.0. it is 18.0."